core/controlplane/config/templates/cloud-config-controller

```diff
@@ -205,6 +205,7 @@ coreos:
         Wants=flanneld.service cfn-etcd-environment.service
         After=cfn-etcd-environment.service
         [Service]
+        EnvironmentFile=/etc/environment
```

Comment:
TODO Comment about its usage https://github.com/kubernetes-incubator/kube-aws/pull/792#discussion_r129494580 (It wasn't very obvious to me)